src/harness/fake_sas.py

```diff
@@ -222,6 +237,18 @@ def do_POST(self):
     self.end_headers()
     self.wfile.write(json.dumps(response))
 
+  def do_GET(self):
+    """Handles GET requests."""
+    path, value = self._parseUrl(self.path)
+    if path == "/v1.0/sas_impl":
```

Comment:
Sorry, you're completely right -- I was misreading do_POST().

I wonder if someday we'll want to version the admin API... but that's a question for later.